utils/create-validate-data-sources.ts

```diff
@@ -267,3 +56,5 @@ const main = async () => {
 if (require.main === module) {
   main();
 }
+
+export default main;
```

Comment:
Are we sure we want to export a default from this file? If it's just being run as a script, there is no need to export the `main` function